components/ui/pathogen-map/pathogen-map-layer.tsx

```diff
@@ -0,0 +1,46 @@
+import { Layer, Source } from "react-map-gl";
```

Comment:
The idea with these `pathogen-map` files is that the arbotracker map and the serotracker map can share the `<PathogenMap>` component and literally every other component here is just something that lives under `<PathogenMap>`. Let me know if you have any questions :+1: